docs/src/modules/components/AppSettingsDrawer.js

```diff
@@ -42,9 +42,21 @@ function AppSettingsDrawer(props) {
   const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');
   const preferredMode = prefersDarkMode ? 'dark' : 'light';
 
+  const updateBodyClass = (currentMode) => {
+    if (currentMode === 'dark') {
+      document.body.classList.remove('mode-light');
+      document.body.classList.add('mode-dark');
+    } else {
+      document.body.classList.remove('mode-dark');
+      document.body.classList.add('mode-light');
+    }
+  };
```

Comment:
I don't think that this logic should be here. This is **one** of the UI controls. The core of the logic is in `/docs/src/modules/components/ThemeContext.js`.